UnityProject/Assets/Scripts/UI/Objects/Cargo/GUI_CargoPageStatus.cs

```diff
@@ -1,65 +1,25 @@
-﻿using System.Collections.Generic;
+using System.Collections;
 using UnityEngine;
 using UI.Core.NetUI;
 using Systems.Cargo;
+using Objects.Cargo;
+
 
 namespace UI.Objects.Cargo
 {
 	public class GUI_CargoPageStatus : GUI_CargoPage
 	{
-		public NetText_label creditsText;
-		public NetText_label shuttleButtonText;
-		public NetText_label messageText;
-		public NetColorChanger statusCargoImage;
-		public NetColorChanger statusTransitImage;
-		public NetColorChanger statusCentcomImage;
-
-		public override void OpenTab()
-		{
-			CargoManager.Instance.OnCreditsUpdate.AddListener(UpdateTab);
-			CargoManager.Instance.OnShuttleUpdate.AddListener(UpdateTab);
-			CargoManager.Instance.OnTimerUpdate.AddListener(UpdateTab);
-		}
+		private string logs;
+		[SerializeField]
+		private NetText_label logLabel;
 
 		public override void UpdateTab()
 		{
-			var cm = CargoManager.Instance;
-
-			if (cm.ShuttleStatus == ShuttleStatus.OnRouteCentcom ||
-				cm.ShuttleStatus == ShuttleStatus.OnRouteStation)
-			{
-				if (cm.CurrentFlyTime > 0)
-				{
-					var min = Mathf.FloorToInt(cm.CurrentFlyTime / 60).ToString();
-					var sec = (cm.CurrentFlyTime % 60).ToString();
-					sec = sec.Length >= 10 ? sec : $"0{sec}";
+			logs = CargoManager.Instance.CentcomMessage;
 
-					shuttleButtonText.MasterSetValue($"{min}:{sec}");
-				}
-				else
-				{
-					shuttleButtonText.MasterSetValue("ARRIVING");
-				}
-				SetShuttleStatus(statusTransitImage);
-			}
-			else
-			{
-				SetShuttleStatus(cm.ShuttleStatus == ShuttleStatus.DockedStation ? statusCargoImage : statusCentcomImage);
-				shuttleButtonText.MasterSetValue("SEND");
-			}
-
-			messageText.MasterSetValue(cm.CentcomMessage);
-			creditsText.MasterSetValue(cm.Credits.ToString());
+			logLabel.SetValue(logs);
```

Comment:
I think logs should be added into, not hard set, so they can match ss13.

Reply:
Its the way the logs are set up in CargoManager, the cargomanager logs add to each other and then I just use that as the input string.

Reply:
understandable